java/maven.indexer/src/org/netbeans/modules/maven/indexer/NexusRepositoryIndexerImpl.java

```diff
@@ -1558,7 +1558,7 @@ public void run(ResultImpl<NBVersionInfo> result) {
                                 }
                             }
                         } else if (ArtifactInfo.ARTIFACT_ID.equals(fieldName)) {
-                            String aid = one.replaceAll("-", "?").replaceAll("\\.", "?");
+                            String aid = one.replace("-", "?").replaceAll("\\.", "?");
```

Comment:
The \\. conversion does not seem correct.

Reply:
The "\." replaceAll() is untouched. Only the "-" -> "?" replaceAll() was changed.

Reply:
Yes, sorry, I missed that.